file.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: false
+
+# first method bubble sort
+
+def bubble_sort(arr)
+  arr.sort!
+end
```

Comment:
**[REQUIRED]**
* We should not use `#sort` method to implement bubble sort algorithm.
* We must implement the correct methodology for the bubble sort algorithm. Check out [this](https://www.geeksforgeeks.org/bubble-sort/). Check this too :point_down: 
![rec](https://user-images.githubusercontent.com/44072711/69215260-321ba200-0b37-11ea-8476-b67b09766c13.png)